public/app/plugins/panel/histogram/Histogram.tsx

```diff
@@ -49,12 +50,16 @@ export interface HistogramProps extends Themeable2 {
 
 export function getBucketSize(frame: DataFrame) {
   // assumes BucketMin is fields[0] and BucktMax is fields[1]
-  return frame.fields[0].type === FieldType.string ? 1 : frame.fields[1].values[0] - frame.fields[0].values[0];
+  return frame.fields[0].type === FieldType.string
```

Comment:
Why 9 decimal places?

Reply:
not too many, not too few. :shrug: 

![image](https://github.com/grafana/grafana/assets/43234/50939a28-4ee3-485d-a366-66ea870b499b)